app/scripts/modules/amazon/src/serverGroup/configure/serverGroupConfiguration.service.ts

```diff
@@ -120,6 +133,7 @@ export class AwsServerGroupConfigurationService {
     'GroupTotalInstances',
   ];
   private healthCheckTypes = ['EC2', 'ELB'];
+  private volumeTypes = ['gp2', 'st1', 'io1', 'sc1'];
```

Comment:
```suggestion
  private volumeTypes = ['gp2', 'gp3', 'st1', 'io1', 'io2', 'sc1'];
```

`gp3` is the new general-purpose volume type from AWS. This is only available if you're using LaunchTemplates. 
`io2` is their new provisioned IO volume type, only available in preview currently on r5b instances.
 (see https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volume-types.html)